docs/api/security.rst

```diff
@@ -22,40 +22,56 @@ Authorization API Functions
 Constants
 ---------
 
+.. attribute:: NO_PERMISSION_REQUIRED
+
+	A special permission which indicates that the view should always
+	be executable by entirely anonymous users, regardless of the
+	default permission, bypassing any :term:`authorization policy`
+	that may be in effect.  Its actual value is the string
+	'__no_permission_required__'.
```

Comment:
```suggestion
	``'__no_permission_required__'``.
```

Reply:
duplicate

Reply:
It’s because I have done the same changes to docs in pyramid.security and pyramid.authorization :slightly_smiling_face: